packages/components/src/tools-panel/test/index.js

```diff
@@ -172,16 +172,17 @@ const getMenuButton = () => {
  * @return {HTMLElement} The menuButton.
  */
 const openDropdownMenu = async () => {
+	const user = userEvent.setup();
```

Comment:
The return type annotation is wrong, triggering TS warnings on the usages with await ("'await' has no effect on the type of this expression").

```suggestion
 */
const openDropdownMenu = async () => {
	const user = userEvent.setup();
```

or

```suggestion
 * @return {Promise<HTMLElement>} The menuButton.
 */
const openDropdownMenu = async () => {
	const user = userEvent.setup();
```

Reply:
That makes sense, thanks, @mirka! 🙌